pandas/tests/indexes/numeric/test_numeric.py

```diff
@@ -327,7 +327,7 @@ def test_constructor_from_list_no_dtype(self):
         index = Index([1, 2, 3])
         assert index.dtype == np.int64
 
-    def test_constructor(self, dtype):
+    def test_constructor(self, dtype, using_copy_on_write):
```

Comment:
```suggestion
    def test_constructor(self, dtype):
```

(unused after the edits)